planning/autoware_behavior_velocity_planner/launch/behavior_velocity_planner.launch.xml

```diff
@@ -28,9 +28,9 @@
   <arg name="behavior_velocity_planner_no_drivable_lane_module_param_path"/>
   <arg name="behavior_velocity_planner_dynamic_obstacle_stop_module_param_path"/>
   <!-- <arg name="behavior_velocity_planner_template_module_param_path"/> -->
-  <arg name="behavior_velocity_planner_param_file" default="$(find-pkg-share behavior_velocity_planner)/config/behavior_velocity_planner.param.yaml"/>
+  <arg name="behavior_velocity_planner_param_file" default="$(find-pkg-share autoware_behavior_velocity_planner)/config/behavior_velocity_planner.param.yaml"/>
 
-  <node pkg="behavior_velocity_planner" exec="behavior_velocity_planner_node" name="behavior_velocity_planner" output="screen">
+  <node pkg="autoware_behavior_velocity_planner" exec="autoware_behavior_velocity_planner_node" name="autoware_behavior_velocity_planner" output="screen">
```

Comment:
```suggestion
  <node pkg="autoware_behavior_velocity_planner" exec="autoware_behavior_velocity_planner_node" name="behavior_velocity_planner" output="screen">
```
https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/ros-nodes/class-design/#gnss_poser_nodecpp

The default node name shouldn't have `autoware_` prefix.

Reply:
@esteve I think this is all from my side. Then we can merge.